packages/website/pages/new-file.js

```diff
@@ -67,12 +70,15 @@ export default function NewFile() {
             setPercentComplete(Math.round((totalBytesSent / car.size) * 100))
           },
         })
-      } catch (err) {
+
+        router.push({ pathname: '/files', query: version ? { version } : null })
+        setError('')
```

Comment:
Can we clear the error before we start uploading, so that on retry we don't have a stale error message?

Reply:
Sure, it now makes sense since we don't autoclear it :)